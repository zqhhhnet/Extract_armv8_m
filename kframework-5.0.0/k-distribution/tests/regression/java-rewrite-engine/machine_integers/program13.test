svalueMInt(andMInt(lshrMInt(mi(32, -1), 29), shlMInt(mi(32, 3), 1)))

