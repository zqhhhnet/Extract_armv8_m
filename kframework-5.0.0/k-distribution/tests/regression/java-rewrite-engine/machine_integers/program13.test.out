<k>
    6
</k>
